src/redis-cli.c

```diff
@@ -46,8 +46,8 @@
 #include <limits.h>
 #include <math.h>
 
-#include "hiredis.h"
-#include "sds.h"
+#include <hiredis.h>
+#include <sds.h> /* use sds.h from hiredis, so that only one set of sds functions will be present in the binary */
```

Comment:
with the previous code (include with double quotes), the result was that redis-cli uses the inline functions of sds from src/sds.h but uses the non-inline functions of sds from the sds.c implementation inside hiredis.

now that these implementations are not the same (different sds headers), this will no longer work!
this change causes redis-cli to use the sds inline functions from hiredis.